packages/trader/src/Modules/Contract/Containers/contract-replay.tsx

```diff
@@ -186,10 +168,41 @@ const ContractReplay = observer(({ contract_id }: { contract_id: number }) => {
                                         is_reset_contract={is_reset_contract}
                                     />
                                 )}
-                            </MobileWrapper>
+                            </div>
+                        </React.Suspense>
+                    </Div100vhContainer>
+                </PageOverlay>
+            </FadeWrapper>
+        );
+    }
+
+    return (
+        <FadeWrapper is_visible={is_visible} className='contract-details-wrapper' keyname='contract-details-wrapper'>
+            <PageOverlay
+                id='dt_contract_replay_container'
+                header={<Localize i18n_default_text='Contract details' />}
+                onClickClose={onClickClose}
+            >
+                <div className='trade-container__replay'>
```

Comment:
previously <Div100vhContainer was here for desktop instead of this <div. please elaborate why we need to remove <Div100vhContainer.

Reply:
The previous code was using <Div100vhContainer with is_disabled={isDesktop()} prop. So, i suppose its not being used in desktop view. Here i separate mobile and desktop code but i still need `trade-container__replay` class. So i changed it to div